### PR TITLE
Update Empyrion - Custom Scenario and Default Backup Exclusions

### PR DESCRIPTION
**Pull request by IceOfWraith**
This provides a second scenario input that allows the user to specify a custom scenario. The folder name must match the name put in the input.

This also includes a backup exclusion that only stores the Saves folder and the yaml config files.

### CODE REVIEW

File: empyrion-galactic-survivalconfig.json [hunk 1]
```diff
@@ -353,7 +353,7 @@
     "EnumValues": {}
   },
   {
-    "DisplayName": "Custom Scenario",
+    "DisplayName": "Scenario",
     "Category": "Emp - Game Settings",
     "Description": "Choose from premade scenarios.",
```
Comment (Greelan) [inline empyrion-galactic-survivalconfig.json:358]: Maybe note here that that name of the scenario needs to be set in the Custom Scenario setting?

File: empyrion-galactic-survivalupdates.json [hunk 2]
```diff
@@ -13,5 +13,13 @@
...
-  }
+  },
+  {
+    "UpdateStageName":"Backup Exclusions Download 1",
+    "UpdateSourcePlatform":"All",
+    "UpdateSource":"FetchURL",
+    "UpdateSourceData":"https://github.com/IceOfWraith/AMPTemplates/raw/dev/exclusions/.backupExclusionsEmpyrion1",
+    "UpdateSourceArgs":".backupExclude",
+    "UpdateSourceTarget":"{{$FullBaseDir}}",
+    "OverwriteExistingFiles":false}
```
Comment (Greelan) [inline empyrion-galactic-survivalupdates.json:24]: Fix your JSON, bud. :)

File: exclusions/.backupExclusionsEmpyrion1 [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+Empyrion_Example.props
+EmpyrionDedicated.cmd
+EmpyrionDedicated_NoGraphics.cmd
+EmpyrionLauncher.exe
+steamclient.dll
+steamclient64.dll
+tier0_s.dll
+tier0_s64.dll
+vstdlib_s.dll
+vstdlib_s64.dll
```
Comment (Greelan) [inline exclusions/.backupExclusionsEmpyrion1:17]: I will take your word for it that these are the appropriate files to exclude.

Comment (IceOfWraith) [reply exclusions/.backupExclusionsEmpyrion1:17]: Hopefully, it's correct. :)

**Comment by Greelan** [comment]
Looks good. Update your repo path and good to go

**Review by Greelan — APPROVED** [review]

**Pull request merged**